src/libraries/System.Globalization/tests/System.HybridGlobalization.Tests.csproj

```diff
@@ -0,0 +1,10 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFramework>$(NetCoreAppCurrent)</TargetFramework>
+    <TestRuntime>true</TestRuntime>
+    <HybridGlobalization>true</HybridGlobalization>
+  </PropertyGroup>
+  <ItemGroup>
+    <Compile Include="System\Globalization\TextInfoTests.cs" />
+  </ItemGroup>
```

Comment:
I guess we can add HybridMode related tests under https://github.com/dotnet/runtime/tree/main/src/libraries/System.Globalization/tests/Hybrid, and to not mix for now we can have separate WASM/ OSX folders.

Reply:
Done